CONTRIBUTING.rst

```diff
@@ -35,7 +35,7 @@ If you have a database dump, you can load it into your PostgreSQL database by ru
 Note that your database container needs to be running while you do that. You
 can start just the postgresql container by runnin::
 
-  $ docker-compose run postgresql
+  $ docker-compose -d run postgresql
```

Comment:
That's not going to work, the `-d` must be at the end.

Reply:
Whoops!